### PR TITLE
updates Twitter search Keyword field to Query field

### PR DESCRIPTION
**Pull request by TovaCohen**
This updates the twitter.json file to change the name of the Keyword input field to Query. 
It also updates the hint to: 
Return tweets that match this Twitter-format search expression.

### CODE REVIEW

File: app/connector/twitter/src/main/resources/META-INF/syndesis/connector/twitter.json [hunk 1]
```diff
@@ -77,11 +77,11 @@
               "keywords": {
                 "componentProperty": false,
                 "deprecated": false,
-                "displayName": "Keywords",
+                "displayName": "Query",
                 "group": "common",
                 "javaType": "java.lang.String",
                 "kind": "path",
-                "labelHint": "Multiple search values can be separated with comma.",
+                "labelHint": "Return tweets that match this Twitter-format search expression.",
```
Comment (claudio4j) [inline app/connector/twitter/src/main/resources/META-INF/syndesis/connector/twitter.json]: How can the user know what is the "Twitter-format" ? WDYT to add an example: camel OR knative ?

Comment (TovaCohen) [reply app/connector/twitter/src/main/resources/META-INF/syndesis/connector/twitter.json]: I did a web search for "Twitter-format search operator" and easily found the info. I don't want to write something that implies that that is the only operator you can specify. But you're right, examples are always helpful. How about: 
Twittter-format search expression, for example: "My Product" returns tweets that contain exactly My Product.

Comment (claudio4j) [reply app/connector/twitter/src/main/resources/META-INF/syndesis/connector/twitter.json]: That sounds good, thanks Tova.

**Comment by TovaCohen** [comment]
@claudio4j  I committed and pushed that update to this PR. Can you review and merge if correct? Thanks!

**Pull request merged**